introduction_to_amazon_algorithms/lda_topic_modeling/LDA-Introduction.ipynb

```diff
@@ -117,15 +118,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {
-    "collapsed": true,
     "isConfigCell": true,
     "tags": [
      "parameters"
     ]
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Couldn't call 'get_role' to get Role ARN from role name hongshan to get Role path.\n"
```

Comment:
Definitely don't want this kind of thing in there...

Reply:
cleared in this commit b59d325